openedx/core/djangoapps/video_config/toggles.py

```diff
@@ -15,3 +15,14 @@
 PUBLIC_VIDEO_SHARE = CourseWaffleFlag(
     f'{WAFFLE_FLAG_NAMESPACE}.public_video_share', __name__
 )
+
+# .. toggle_name: video_config.transcript_feedback
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Gates access to the transcript feedback widget feature.
+# .. toggle_use_cases: temporary, opt_in
+# .. toggle_creation_date: 2023-05-10
+# .. toggle_target_removal_date: None
+TRANSCRIPT_FEEDBACK = CourseWaffleFlag(
```

Comment:
How are waffle flags toggled on and off in production and staging?

Reply:
That is correct, I will add them into CMS Django Admin for Stage and Prod (and Edge, as needed).